src/converters/webrtc.h

```diff
@@ -620,6 +623,48 @@ struct Converter<webrtc::MediaStreamTrackInterface::TrackState, std::string> {
   static Validation<std::string> Convert(webrtc::MediaStreamTrackInterface::TrackState value);
 };
 
+template <>
+struct Converter<node_webrtc::MediaStream*, v8::Local<v8::Value>> {
+  static Validation<v8::Local<v8::Value>> Convert(node_webrtc::MediaStream* track) {
+    Nan::EscapableHandleScope scope;
+    if (!track) {
+      return Validation<v8::Local<v8::Value>>::Invalid("MediaStream is null");
+    }
+    return Validation<v8::Local<v8::Value>>::Valid(scope.Escape(track->handle()));
+  }
+};
+
+template <>
+struct Converter<v8::Local<v8::Value>, node_webrtc::MediaStream*> {
+  static Validation<node_webrtc::MediaStream*> Convert(v8::Local<v8::Value> value) {
+    // TODO(mroberts): This is not safe.
+    return value->IsObject() && !value->IsArray()
+        ? Validation<node_webrtc::MediaStream*>::Valid(Nan::ObjectWrap::Unwrap<node_webrtc::MediaStream>(value->ToObject()))
+        : Validation<node_webrtc::MediaStream*>::Invalid("IDK");
+  }
+};
+
+template <>
+struct Converter<node_webrtc::MediaStreamTrack*, v8::Local<v8::Value>> {
+  static Validation<v8::Local<v8::Value>> Convert(node_webrtc::MediaStreamTrack* track) {
+    Nan::EscapableHandleScope scope;
+    if (!track) {
+      return Validation<v8::Local<v8::Value>>::Invalid("MediaStreamTrack is null");
+    }
+    return Validation<v8::Local<v8::Value>>::Valid(scope.Escape(track->handle()));
+  }
+};
+
+template <>
+struct Converter<v8::Local<v8::Value>, node_webrtc::MediaStreamTrack*> {
+  static Validation<node_webrtc::MediaStreamTrack*> Convert(v8::Local<v8::Value> value) {
+    // TODO(mroberts): This is not safe.
+    return value->IsObject() && !value->IsArray()
+        ? Validation<node_webrtc::MediaStreamTrack*>::Valid(Nan::ObjectWrap::Unwrap<node_webrtc::MediaStreamTrack>(value->ToObject()))
+        : Validation<node_webrtc::MediaStreamTrack*>::Invalid("IDK");
+  }
+};
```

Comment:
The safe way to do this, and the conversion above, would be something like this: https://stackoverflow.com/questions/8994196/how-to-check-for-correct-type-when-calling-objectwrapunwrap-in-a-nodejs-add-on